consumers/writers/influxdb/consumer.go

```diff
@@ -29,17 +31,21 @@ type RepoConfig struct {
 }
 
 type influxRepo struct {
-	client influxdb2.Client
-	cfg    RepoConfig
-	async  bool
+	client           influxdb2.Client
+	cfg              RepoConfig
+	async            bool
+	writeAPI         api.WriteAPI
+	writeAPIBlocking api.WriteAPIBlocking
 }
 
 // New returns new InfluxDB writer.
 func New(client influxdb2.Client, config RepoConfig, async bool) consumers.Consumer {
 	return &influxRepo{
-		client: client,
-		cfg:    config,
-		async:  async,
+		client:           client,
+		cfg:              config,
+		async:            async,
+		writeAPI:         client.WriteAPI(config.Org, config.Bucket),
```

Comment:
Let's support only blocking API until #1742 is merged and we'll add async support later.